### ISSUE TITLE
What's up with the IDs on 0x10^c hardware?

### ISSUE THREAD

**Issue by azertyfun** [techcompliant/TC-Specs#15]
Old 0x10^c hardware has two IDs documented, one for "older models" and one for "newer models" (see [LEM1802.txt](https://github.com/techcompliant/TC-Specs/blob/master/LEM1802.txt)). Is it to comply with ID specs? Or is it something else? Which one is in use in game?

**Comment by Meisaka** [comment]
Yes, this is in fact related to getting these items to match up with the Device-ID method. The game and emulators are currently using the "older models" ID, in which the attribute bits are "wrong". I didn't want to replace the full IDs outright because that would break things quite a bit.

Depending on how impractical the idea of attribute bits is, it might make sense to drop them and revert this. This maybe requires a bit of discussion, although I am more in favor of keeping it, in which case software can AND the high word of the ID with 0xFFF0 before comparing to a fixed list (ignoring the bits that are different in this).

**Comment by paultech** [comment]
What's our status here @Meisaka ?

**Comment by Meisaka** [comment]
I was hoping to get #16 merged, and change one of the bits to indicate the presence of an EEPROM, at which point these kind of changes would be more finalized.
